google_photos_takeout_helper/__main__.py

```diff
@@ -744,9 +744,9 @@ def _walk_with_tqdm(res, bar: _tqdm):
             logger.info(f" - you have full list in {f.name}")
 
     logger.info('')
-    logger.info('Sooo... what now? You can see README.md for what nice G Photos alternatives I found and recommend')
+    logger.info('Sooo... what now? You can see README.md for what nice G Photos alternatives I have found and would recommend')
     logger.info('')
-    logger.info('If I helped you, you can consider donating me: https://www.paypal.me/TheLastGimbus')
+    logger.info('If I helped you, you can consider donating: https://www.paypal.me/TheLastGimbus')
```

Comment:
Who else then?

Reply:
I believe that you sound less pretentious. Also stating that the donations are going to you seems somewhat redundand

Reply:
That's even better. It makes it clear that I take all the cash, and not "some unnamed organisation"
# :sunglasses: :money_with_wings: